templates/consul-maint-before-shutdown.service.j2

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Consul maintenance task at shutdown
+After=consul.service
+After={{ consul_service_name }}.service
```

Comment:
In case the value for `consul_service_name` is wrong, I would prefer this not to fail silently. Will it fail at the `state=started` task?

If not, could we have some alert? Like an `assert` that the service file exists, which will fail the whole recipe and send a Mattermost alert?

Also, this implies that the Consul service is installed *after* the service it requires, not sure it's the case in every role...

Reply:
Do you know what happens if `consul_service_name` is incorrect?